packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -220,20 +226,63 @@ describe("TableComponent", () => {
 			expect(component.isMultiSortEnabled).toBe(false);
 		});
 
-		it("should assign right value to isMultiSelectEnabled when multiSelect changes and adapt displayedColumns", () => {
-			spyOn(component.displayedColumns, "unshift");
+		it("should assign right value to isMultiSelectEnabled when multiSelect changes and rowsSelectable is enabled", () => {
 			hostComponent.rowsSelectable = true;
 			hostComponent.multiSelect = "true";
 			hostFixture.detectChanges();
 			expect(component.isMultiSelectEnabled).toBe(true);
+
+			hostComponent.multiSelect = "false";
+			hostFixture.detectChanges();
+			expect(component.isMultiSelectEnabled).toBe(false);
+		});
+
+		it("should assign right value to display/hide 'select' column when rowsSelectable changes", () => {
+			spyOn(component.displayedColumns, "unshift").and.callThrough();
+			hostComponent.rowsSelectable = true;
+			hostFixture.detectChanges();
+
 			expect(component.displayedColumns.unshift).toHaveBeenCalledTimes(1);
 			expect(component.displayedColumns.unshift).toHaveBeenCalledWith("select");
+			let rowThElements = <NodeListOf<HTMLElement>>hostFixture.nativeElement.querySelectorAll(tableThSelector);
+			expect(rowThElements.length).toBeGreaterThan(0);
+			let selectThElement = find(rowThElements, (thElement: HTMLElement) => thElement.className.indexOf("cdk-column-select") > -1);
+			expect(selectThElement).toBeDefined();
 
 			(<Spy>component.displayedColumns.unshift).calls.reset();
-			hostComponent.multiSelect = "false";
+			hostComponent.rowsSelectable = false;
 			hostFixture.detectChanges();
-			expect(component.isMultiSelectEnabled).toBe(false);
 			expect(component.displayedColumns.unshift).not.toHaveBeenCalled();
+			rowThElements = <NodeListOf<HTMLElement>>hostFixture.nativeElement.querySelectorAll(tableThSelector);
+			expect(rowThElements.length).toBeGreaterThanOrEqual(0);
+			selectThElement = find(rowThElements, (thElement: HTMLElement) => thElement.className.indexOf("cdk-column-select") > -1);
+			expect(selectThElement).toBeUndefined();
+		});
+
+		it("should assign right value to _showRowIndex when showRowIndex changes and adapt displayedColumns", () => {
+			spyOn(component.displayedColumns, "unshift").and.callThrough();
```

Comment:
Can you remove this line from this test? As I mentioned before, we should avoid testing these kind of "internals"